src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
I'm fine with it here. It's the coding style we have for everything here.

@amphp uses PSR-2 with the exception that braces must be on the same line.